iModelCore/ecobjects/src/SchemaMerger.cpp

```diff
@@ -352,6 +352,19 @@ BentleyStatus SchemaMerger::MergeSchemas(SchemaMergeResult& result, bvector<ECSc
         {
         DumpSchemasToFile(result.GetResults(), dumpLocation.c_str(), "Result");
         }
+    
+    if(false == options.GetSkipValidation())
+        {
+        for(const auto& schema : result.GetResults())
+            {
+            if(!const_cast<ECSchemaP>(schema)->Validate(true))
+                {
+                result.Issues().ReportV(IssueSeverity::Error, IssueCategory::BusinessProperties, IssueType::ECSchema, ECIssueId::EC_0010,

```

Comment:
You need to generate a unique IssueId for this. The one used is already in use by ECSchemaConverter